common/src/main/java/io/homeassistant/companion/android/database/AppDatabase.kt

```diff
@@ -193,7 +194,8 @@ abstract class AppDatabase : RoomDatabase() {
                     MIGRATION_21_22,
                     MIGRATION_22_23,
                     MIGRATION_23_24,
-                    Migration40to41(context.assets)
+                    Migration40to41(context.assets),
+                    MIGRATION_45_46
```

Comment:
You only need the `AutoMigration` and version increase in the line at the top. This line and your changes below are a manual migration, please remove them.

Reply:
Good catch! Done.